src/index.tsx

```diff
@@ -1,4 +1,9 @@
 import { createRoot } from 'react-dom/client';
+import { BrowserRouter as Router } from 'react-router-dom';
 import { App } from './App';
 
-createRoot(document.getElementById('root') as HTMLElement).render(<App />);
+createRoot(document.getElementById('root') as HTMLElement).render(
+  <Router>
+    <App />
+  </Router>,
+);
```

Comment:
Consider using React.StrictMode for wrapping your app during development. It helps to identify potential problems in an application and does not render any visible UI. It activates additional checks and warnings for its descendants.